VimeoNetworking/Sources/Models/VIMNotificationsConnection.m

```diff
@@ -17,6 +17,17 @@ @interface VIMNotificationsConnection ()
 
 @implementation VIMNotificationsConnection
 
+- (void)didFinishMapping
+{
+    // This is a migration for the outage on 2/20/18 where these counts were being returned as empty arrays for some users.
+    // https://vimean.atlassian.net/browse/VIM-5996 [ghking] 2/22/18
+    
+    if ([self.type_count isKindOfClass:[NSArray class]])
```

Comment:
instead of just checking to see if this is an NSArray, should we instead verify that self.type_count is an NSDictionary and always create the empty dictionary object if it's not?

Reply:
I'm down with that